concepts/pointers/.meta/config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "blurb": "In C++, pointers are variables that store memory addresses. They allow you to directly manipulate memory and access to data structures like arrays and objects.",
```

Comment:
That looks a bit long. Does it fit wherever it is displayed on the website? Can you shorten it?

Reply:
It's less than 180 chars, I think that is still in the okay range?

> "blurb: A short description of this concept. Its length must be <= 350. Markdown is not supported (required)"
From: https://exercism.org/docs/building/tracks/concepts#h-file-meta-config-json